DQM/SiPixelPhase1TrackClusters/interface/SiPixelPhase1TrackClusters.h

```diff
@@ -30,13 +31,16 @@ class SiPixelPhase1TrackClusters : public SiPixelPhase1Base {
     NTRACKS_VOLUME
   };
 
+  bool ApplyVertexCut_;
```

Comment:
please start variables with lower case (and can this not be a private data member?)